src/modules/DNS.rb

```diff
@@ -519,6 +519,27 @@ def IsHostLocal(check_host)
 
   private
 
+    # Get current hostname and IP Address
+    # if these are set by DHCP
+    # @return map with ip, hostname_short and hostname_fq keys
+    def GetDHCPHostnameIP
```

Comment:
Since it is a private method, it could be renamed in a more Ruby style. Something like `dhcp_data` sounds good for me.